server/src/modules/users/user.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  CreateDateColumn,
+  UpdateDateColumn,
+  JoinColumn,
+} from 'typeorm';
+import { SubscriptionPlanEntity } from '../subscription-plans/subscription-plan.entity';
+
+@Entity({ name: 'users' })
+export class UserEntity {
+  @ApiProperty()
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @ApiProperty({ maxLength: 32 })
+  @Column({ length: 32 })
+  name: string;
+
+  @ApiProperty({ maxLength: 16 })
+  @Column({ length: 16 })
+  password: string;
```

Comment:
exclude password from model with @Exclude decorator and hide this property using @ApiHideProperty instead of @ApiProperty